acrclient/client.py

```diff
@@ -155,3 +156,14 @@ def get_bm_cs_projects_results(
             params=params,
             **kwargs,
         ).get("data")
+
+    def get_external_metadata_tracks(
+        self, params: Optional[GetExternalMetadataTracksParams] = None
+    ) -> list:
+        try:
+            return self.json(path="/api/external-metadata/tracks", params=params).get(
+                "data"
+            )
+        except HTTPError as error:
```

Comment:
Is this error persistent? if it's a temporary error a retry/backoff loop might be better than ignoring it.

Reply:
:thinking: retry/backoff code is already here:
https://github.com/radiorabe/python-acrclient/blob/399bcc9adac4e6b5c290f6672f6fff3f76083fd1/acrclient/client.py#L46-L56

maybe we need to tune it, or it doesn't include 500 errors

Reply:
For some entries, the error persists and for some it returns 200 after retrying a few seconds later. You're absolutely correct to handle this in the central location.